components/builder-api-proxy/default.toml

```diff
@@ -1,47 +1,35 @@
-app_url              = "https://bldr.habitat.sh"
-community_url        = "https://www.habitat.sh/community"
+cookie_domain        = ""
+demo_app_url         = "https://www.habitat.sh/demo/build-system/steps/1/"
 docs_url             = "https://www.habitat.sh/docs"
+enable_builder       = false
 environment          = "production"
-friends_only         = false
+hosted               = false
 source_code_url      = "https://github.com/habitat-sh/habitat"
-tutorials_url        = "https://www.habitat.sh/tutorials"
-www_url              = "https://www.habitat.sh"
 status_url           = "https://status.habitat.sh/"
-forums_url           = "https://forums.habitat.sh/"
-events_url           = "https://events.chef.io/events/categories/habitat/"
-roadmap_url          = "https://ext.prodpad.com/ext/roadmap/d2938aed0d0ad1dd62669583e108357efd53b3a6"
-feature_requests_url = "https://portal.prodpad.com/24539"
-slack_url            = "http://slack.habitat.sh/"
-youtube_url          = "https://www.youtube.com/playlist?list=PL11cZfNdwNyOxlvI1Kq6ae8eVBl5S3IKk"
-demo_app_url         = "https://www.habitat.sh/demo"
-learn_url            = "https://www.habitat.sh/learn"
-enable_builder       = true
-enable_access_tokens = true
+tutorials_url        = "https://www.habitat.sh/learn"
 use_gravatar         = true
-hosted               = false
-cookie_domain        = ""
+www_url              = "https://www.habitat.sh"
 
 [github]
-url          = "https://api.github.com"
-web_url      = "https://github.com"
-app_url      = "https://github.com/apps/habitat-builder"
 app_id       = 5565
+app_url      = "https://github.com/apps/habitat-builder"
+url          = "https://api.github.com"
 
 [oauth]
-provider       = "github"
-client_id      = ""
 authorize_url  = "https://github.com/login/oauth/authorize"
+client_id      = ""
```

Comment:
wierd that a bunch of lines are just getting re-ordered..

Reply:
Yep -- I noted in the PR description that I reordered these to alphabetize them, just to make things easier to scan for.

Reply:
👍